alerta/views/oembed.py

```diff
@@ -9,28 +9,21 @@
 from alerta.models.enums import Scope
 from alerta.utils.response import jsonp
 
+from collections import namedtuple
+
 from . import api
 
+Query = namedtuple('Query', ['where', 'vars', 'sort', 'group'])
 
 @api.route('/oembed', defaults={'format': 'json'}, methods=['OPTIONS', 'GET'])
 @api.route('/oembed.<format>', methods=['OPTIONS', 'GET'])
 @cross_origin()
 @permission(Scope.read_oembed)
 @jsonp
 def oembed(format):
-
-    if format != 'json':
-        return jsonify(status='error', message='unsupported format: %s' % format), 400
-
-    if 'url' not in request.args or 'maxwidth' not in request.args \
-            or 'maxheight' not in request.args:
-        return jsonify(status='error', message='missing default parameters: url, maxwidth, maxheight'), 400
-
     try:
         url = request.args['url']
-        width = int(request.args['maxwidth'])
-        height = int(request.args['maxheight'])
-        title = request.args.get('title', 'Alerts')
+        title = request.args['title']
```

Comment:
Why did you remove the default title? It will throw an exception here now if someone does not provide `title` as a URL param.